search-app/docs/api.md

```diff
@@ -5405,3 +5412,152 @@ Example un-healthy response body:
+
+__Example__
+
+```
+curl -v -XPOST -H "Authorization: Bearer XXXXX" -H "Content-Type:application/json" "%CMR-ENDPOINT%/associate/GRD1200000000-PROV1" -d '[{"concept_id": "S1200000001-PROV1"}]'
+```
+__Example Response__
+```
+[{
+  "generic_association" : {
+    "concept_id" : "GA1200000000-CMR",
+    "revision_id" : 3
+  },
+  "associated_item" : {
+    "concept_id" : "S1200000001-PROV1"
+  }}]
+```
+
+#### <a name="generic-multi-association"></a> Associate Multiple Concepts Simultaneously
+Multiple associaitons with or without association details can be made to a specified concept on a single request by passing multiple concept ids in a list form. The response returns the association cocncept and the concept being associated.
+__Example__
+```
+curl -v -XPOST -H "Authorization: Bearer XXXXX" -H "Content-Type:application/json" "%CMR-ENDPOINT%/associate/GRD1200000006-PROV1" -d '[{"concept_id":"S1200000008-PROV1"},{"concept_id": "C1200000007-PROV1"}]'
+```
+__Example Response__
+```
+[{"generic_association":{"concept_id":"GA1200000009-CMR","revision_id":3},"associated_item":{"concept_id":"S1200000008-PROV1"}},{"generic_association":{"concept_id":"GA1200000010-CMR","revision_id":3},"associated_item":{"concept_id":"C1200000007-PROV1"}}]%
+```
+#### <a name="generic-association-with-revisions"></a>Associate Concepts by Revision Id
+An optional "revision_id" field can be passed into the association to create an association to specific revisions of two concepts.
+__Example__
+
+```
+curl -v -XPOST -H "Authorization: Bearer XXXXX" -H "Content-Type:application/json" "%CMR-ENDPOINT%/associate/GRD1200000011-PROV1" -d \ '[{
+        "concept_id": "SO1200000012-PROV1",
+        "revision_id ": 2
+}]'
+
+```
+__Example Response__
+```
+[{"generic_association":{"concept_id":"GA1200000013-CMR","revision_id":1},"associated_item":{"concept_id":"SO1200000012-PROV1"}}]%
+```
+
+#### <a name="generic-association-with-payloads"></a> Associate Concepts With Details
+__Example__
+```
+curl -v -XPOST -H "Authorization: Bearer XXXXX" -H "Content-Type:application/json" "%CMR-ENDPOINT%/associate/GRD1200000000-PROV1" -d '[{"concept_id": "S1200000001-PROV1", "data": {"field": "value", "another_field": {"XYZ": "ZYX"}}}]'
```

Comment:
-v

Reply:
I had also interpreted "payload" as an internal working title, I kept it part of the markdown thinking it was useful but, it isn't used in the actual readable documentation. I can change it to data though to make everything a bit more consistent